xblockmufi/private/edit.js

```diff
@@ -20,8 +20,10 @@ function XblockMufiEdit(runtime, element) {
         $.ajax(url, {
             type: 'POST',
             data: JSON.stringify({
-                // TODO: Add entries here for each field to be saved
-                'name': $('#xblock_xblockmufi_name').val(),
+                'display_name': $('#mufi_edit_display_name').val(),
```

Comment:
Ugh, I should update the SDK to refer to this relatively instead of via ID selectors.